Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -479,12 +483,12 @@
             ]
         },
         {
-            "name": "nodeLayoutContextTypeChanged",
-            "description": "Called when a node's layout context type has changed.",
+            "name": "nodeLayoutFlagsChanged",
+            "description": "Called when the layout of a node changes.",
```

Comment:
`Called when the layout of a node changes in a way that is important to Web Inspector.`?